config/urls.py

```diff
@@ -27,31 +27,32 @@
 from foodsaving.invitations.api import InvitationsViewSet, InvitationAcceptViewSet
 from foodsaving.stores.api import StoreViewSet, PickupDateViewSet, PickupDateSeriesViewSet, FeedbackViewSet
 from foodsaving.subscriptions.api import PushSubscriptionViewSet
-from foodsaving.userauth.api import AuthUserView, AuthView, LogoutView, AuthStatusView
+from foodsaving.userauth.api import AuthUserView, AuthView, LogoutView, VerifyMailView, ResendVerificationView, \
+    ResetPasswordView
 from foodsaving.users.api import UserViewSet
 
 router = routers.DefaultRouter()
 
-router.register(r'groups', GroupViewSet)
-router.register(r'groups-info', GroupInfoViewSet)
-router.register(r'agreements', AgreementViewSet)
+router.register('groups', GroupViewSet)
```

Comment:
Oh, wonder what the `r` was for.

Reply:
`r` makes it a [raw literal](https://docs.python.org/3/reference/lexical_analysis.html#string-and-bytes-literals), which commonly used for specifying regular expressions. Not needed here, so I removed it.

(`r` actually not _needed_ for regexp, but it's a [convenience method](https://stackoverflow.com/questions/3517802/python-raw-literal-string) to prevent the need for double escaping stuff)